8-mcp/2-mcp-via-configuration/README.md

```diff
@@ -0,0 +1,17 @@
+# Using tools with an MCP server from NPM with vanilla LangChain4j
+
+## Requirements for running this app
+By default this app uses OpenAI's gpt-4o model.
+
+You need to set the `OPENAI_API_KEY` environment variable to the value of your OpenAI key.
+
+## Running this app
+1. Run `./mvnw clean quarkus:dev`
+2. See the output in the console
+3. Check that a file `playground/sum.py` was created with a Python script that sums two numbers
```

Comment:
I haven't tried this so maybe this comment is irrelevant, but in `application.properties` I see some stuff with npm. Is there any additional setup the user would need to do to get this example to work?

Reply:
So yes, please try running it yourself and let's see how it goes